package.json

```diff
@@ -24,9 +24,11 @@
     "debug": "^2.2.0",
     "ioredis": "^2.2.0",
     "loopback-connector": "^2.4.0",
-    "msgpack5": "^3.4.0"
+    "msgpack5": "^3.4.0",
+    "strong-globalize": "^2.6.6"
   },
   "devDependencies": {
+    "bluebird": "^3.4.1",
```

Comment:
This is the part "fix CI".

KVAO test suite requires `bluebird` to be installed, the same way as it requires a specific version of `should`. See https://github.com/strongloop/loopback-datasource-juggler/pull/1044/files